<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finite Automata</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Graph</string>
    </property>
    <widget class="QMenu" name="menuNode">
     <property name="title">
      <string>Node</string>
     </property>
     <addaction name="actionNew"/>
     <addaction name="actionChange_state"/>
     <addaction name="actionChange_name"/>
     <addaction name="actionDelete"/>
    </widget>
    <widget class="QMenu" name="menuConnection">
     <property name="title">
      <string>Conection</string>
     </property>
     <addaction name="actionConnect"/>
     <addaction name="actionDisconnect"/>
    </widget>
    <addaction name="menuNode"/>
    <addaction name="menuConnection"/>
   </widget>
   <widget class="QMenu" name="menuSolve">
    <property name="title">
     <string>Solve</string>
    </property>
    <widget class="QMenu" name="menuTransform">
     <property name="title">
      <string>Transform</string>
     </property>
     <addaction name="actionDFA_to_Regular_Expression"/>
     <addaction name="actionNFA_to_DFA"/>
     <addaction name="actionEpsilon_NFA_to_DFA"/>
    </widget>
    <widget class="QMenu" name="menuProperties">
     <property name="title">
      <string>Properties</string>
     </property>
     <addaction name="actionUnion"/>
     <addaction name="actionIntersection"/>
     <addaction name="actionDifference"/>
     <addaction name="actionComplement"/>
    </widget>
    <addaction name="actionDFA"/>
    <addaction name="actionNFA"/>
    <addaction name="actionEpsilon_NFA"/>
    <addaction name="separator"/>
    <addaction name="menuTransform"/>
    <addaction name="menuProperties"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGraph"/>
   <addaction name="menuSolve"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDFA">
   <property name="text">
    <string>DFA</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNFA">
   <property name="text">
    <string>NFA</string>
   </property>
  </action>
  <action name="actionEpsilon_NFA">
   <property name="text">
    <string>Ɛ-NFA</string>
   </property>
  </action>
  <action name="actionDFA_to_Regular_Expression">
   <property name="text">
    <string>DFA ⇨ Regular Expression</string>
   </property>
  </action>
  <action name="actionNFA_to_DFA">
   <property name="text">
    <string>NFA ⇨ DFA</string>
   </property>
  </action>
  <action name="actionEpsilon_NFA_to_DFA">
   <property name="text">
    <string>Ɛ-NFA ⇨ DFA</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionChange_state">
   <property name="text">
    <string>Change state</string>
   </property>
  </action>
  <action name="actionChange_name">
   <property name="text">
    <string>Change name</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionUnion">
   <property name="text">
    <string>Union</string>
   </property>
  </action>
  <action name="actionIntersection">
   <property name="text">
    <string>Intersection</string>
   </property>
  </action>
  <action name="actionDifference">
   <property name="text">
    <string>Difference</string>
   </property>
  </action>
  <action name="actionComplement">
   <property name="text">
    <string>Complement</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
